src/Directory.Build.props

```diff
@@ -0,0 +1,5 @@
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+     <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Common.props'))\Common.props" />
```

Comment:
Since you're opted into v15-only features already, you could simplify this file by dropping `ToolsVersion` and the namespace and changing this line to

```xml
<Import Project="$([MSBuild]::GetPathOfFileAbove('Common.props')" />
```

(That property function is new to MSBuild 15: https://github.com/Microsoft/msbuild/issues/1277)